### DISCUSSION TITLE
keyboard input not working for bursts of up to 5 seconds

### DISCUSSION THREAD

**Discussion by gliddies** [ppy/osu#18362] [Q&A]
my laptop is a little old (a windows geo-flex netbook) but i don’t find this issue when typing for long periods of time, it only happens when playing osu.

when playing mania, the keyboard will just suddenly stop working, for about 1 to 5 seconds. it’s not lag, as afterwards there’s no burst of many key presses, it just simply starts working again. i’ve tried changing the specific keys for each lane but it still happens so there’s absolutely no issue with specific keys either.

**Comment by bdach** [comment]
please attach video of this, preferably with the key overlay and the frame graph showing (press ctrl-f11 twice for the latter)
